pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -0,0 +1,133 @@
+package org.broadinstitute.dsm.db.dao.mercury;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.db.dao.Dao;
+import org.broadinstitute.dsm.db.dto.mercury.MercuryOrderDto;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.lddp.db.SimpleResult;
+
+@Slf4j
+public class MercuryOrderDao implements Dao<MercuryOrderDto> {
+    public static String SQL_INSERT_MERCURY_ORDER = "insert into mercury_sequencing (order_id, order_date, ddp_participant_id, "
+            + "kit_type_id, barcode) values (?, ?, ?, ?, ?)";
+
+    public static String SQL_GET_KIT_FROM_BARCODE =
+            "SELECT p.ddp_participant_id, accession_number, ddp.instance_name, t.collaborator_sample_id, settings.mercury_order_creator, "
```

Comment:
mercury_order_creator should go int ddp_instance like bsp_group. I would like to have all GP related info in one table, easier for setting things up